bin/run-tests

```diff
@@ -25,7 +26,7 @@ for D in exercises/*; do
         { cd ${D};
           echo "Running tests for ${EXERCISE_NAME}";
           make clean >> /dev/null;
-          if make | grep FAIL: ; then
+          if make CFLAGS='-DRUN_ALL_TESTS -std=c99 -Wall -Wextra -pedantic -Werror ' | grep FAIL: ; then
```

Comment:
I know his is ugly, but fixing it requires touching every make file.  I can if we think it's worth the effort.

Reply:
Eh, I think we should probably touch them all. All we need to do is change the `=` to a `+=` on the first line. I think this is better than scratching our heads later.

Reply:
Do you have access to a Windows machine to test this on before I go change everything?

Reply:
No. @patricksjackson?

Reply:
I agree, the first line of the default makefile should be updated to `CFLAGS += -std=c99` anyways. The fast way (in case you were planning on doing it manually):
```
find . -name makefile -print0 | xargs -0 sed -i 's/CFLAGS  =/CFLAGS +=/g'
```

For changes to the Makefiles / scripts, if it runs on Ubuntu/Other Linux just assume it works on Windows. We should just push people to use the WSL for that and that is literally Ubuntu LTS (or SUSE, or Fedora (soon), and hopefully others in future).

The shell scripts are an issue (with the repo checked out in Windows they have CRLF line endings, and the shell programs don't like that), but that's just a matter of changing the line endings with `dos2unix`. `make` though works on CRLF files, so they can use it extremely easily.

It's probably on me to make a big README update for Windows before the track launches / soon.